src/CollectionTrait.php

```diff
@@ -164,19 +164,19 @@ protected function _shorten_ml(string $desired): string
         if (
             isset($this->_appScopeTrait) &&
             isset($this->app->max_name_length) &&
-            strlen($desired) > $this->app->max_name_length
+            mb_strlen($desired) > $this->app->max_name_length
```

Comment:
is `max_name_length` for UTF-8 size? (I would say not for the indended behaviour - like shortening keys for browser, but I think it is ok)

Also note, that mb_xxx functions are about 10x slower.

Reply:
well... names can theoretically be anything so can contain utf8 chars.

Reply:
I would bet the hard limitations like in suhosin (I do not use it) are for bytes, but I support modern tech as much as possible, to I am perfectly fine with this